### PR TITLE
67. update the node runners private node creation page colours and skin tones

### LINKED ISSUE #67: Update the node-runner's private node creation page colours and skin tones (Dark mode)
## 📄 Context
Here are the `node runners private node creation` page changes on colour, skin tones and fonts in Dark mode to comply with the new Cartesi brand. The application's structure did not change, only the look and feel.

**PS: Refer to the parent of this issue for references.**
PS²: Don't worry about the **header** and **footer**, because that is addressed in a separate issue #62 

### Overall
- [x] Make sure the fonts are applied correctly based on the Figma description. i.e.  Where to use each font and what size and weight to be applied.
- [x] Make sure the components are round, like the design. **We're transitioning from a squared-based design to a round one.**
### Page sessions
- [x] Apply new colours and font rules for the page header area, also:
  - [x] Apply the new colours and font changes for the `backlink`. 
  - [x] Apply the new colours and font changes for the `page title`. 
  - [x]  Apply the new colours and font changes for the `learn from tutorial` link.
- [x] Apply colour and font changes to the following steps: 
    - [x] For steps involving transactions, apply support system colours for feedback UI, i.e. Alerts, input error messages, spinners, etc.
    - [x] Update the `Set up Ethereum node` step.
    - [x] Update the `Set up node` step.
    - [x] Update the `Hire node` step.
    - [x] Update the `Set allowance` step.
- [x] **On small screens**: Apply the new colours and font rules for the `StepGroup <Header>`. 
- [x] **Add here any extra tasks you see fit [Optional]**

## 🎯 Definition of Done

-   [x] Node runners' private node creation matches the design specs on Figma.

### PR DESCRIPTION
**Pull request by dandheedge**
Fixes #67 #69

### CODE REVIEW

**Review by brunomenezes — CHANGES_REQUESTED** [review]
Hi @dandheedge, 
As you are solving two issues in one PR, here is what I caught on both pages. 

## Private node creation

The first is that links are not using the `dark secondary` colour (teal) you can see the definition on Figma [here](https://www.figma.com/file/WuitYyfH2w4a5tnhXK6aRC/Explorer-design-system?type=design&node-id=2028-6389&mode=design&t=nAfv9Buf5u31BrHX-0) also, by the case stated on Figma the link text should be teal and not cyan (I saw a few). An example on that page is the top link for the tutorial and inside the `Set up Node` box (Reference below).

The second is that the steps counter on the **small devices version** still displays the old colours and tones (reference below)

#### Links
links-not-using-secondary-dark(teal)

#### Step status on small devices
private-node-creation

## Private node management
Things look good. One thing I caught here is that the **allowance** notification/alert is not displaying anymore when the confirmation is sent to the wallet [Reference below]. 

no-allowance-alert

File: apps/staking/src/components/node/NodeHireNodeSection.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ export interface NodeHireNodeSectionProps {
 export const NodeHireNodeSection: FC<NodeHireNodeSectionProps> = (props) => {
     const { isHiring = false, onHire } = props;
     const { account } = useWallet();
-    const bg = useColorModeValue('white', 'gray.800');
+    const bg = useColorModeValue('white', 'rgba(255, 255, 255, 0.06)');
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/NodeHireNodeSection.tsx:41]: If possible, please add a color var for `rgba(255, 255, 255, 0.06)` and reference it from the colors config instead.

File: apps/staking/src/components/node/NodeHireNodeSection.tsx [hunk 2]
```diff
@@ -63,6 +63,8 @@ export const NodeHireNodeSection: FC<NodeHireNodeSectionProps> = (props) => {
             px={{ base: 2, lg: 4, xl: 8 }}
             py={{ base: 2, sm: 4, lg: 8 }}
             mb={6}
+            borderRadius={'2xl'}
+            border="1px solid rgba(255, 255, 255, 0.10)"
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/NodeHireNodeSection.tsx:67]: I think we have a color var for `rgba(255, 255, 255, 0.10)`. Please check the colors config.

File: apps/staking/src/components/node/NodeInfoSection.tsx [hunk 3]
```diff
@@ -66,8 +66,7 @@ export const NodeInfoSection: FC<INodeInfoSection> = ({
     onAuthorize,
 }) => {
     // dark mode support
-    const bg = useColorModeValue('white', 'gray.800');
-    const buttonBg = useColorModeValue('gray.80', 'gray.800');
+    const bgBox = useColorModeValue('white', 'rgba(255, 255, 255, 0.06)');
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/NodeInfoSection.tsx:69]: Please use a color var instead.

File: apps/staking/src/components/node/NodeInfoSection.tsx [hunk 4]
```diff
@@ -103,11 +102,13 @@ export const NodeInfoSection: FC<INodeInfoSection> = ({
             ) : (
                 <>
                     <Box
-                        bg={bg}
+                        border={'1px solid rgba(255, 255, 255, 0.10)'}
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/NodeInfoSection.tsx:105]: Same here.

File: apps/staking/src/components/node/NodeMaturingSection.tsx [hunk 5]
```diff
@@ -34,26 +34,18 @@ export const NodeMaturingSection: FC<INodeMaturingSection> = ({
     maturingBalance,
     maturingLeft = '6 hours',
 }) => {
-    const bg = useColorModeValue('white', 'gray.800');
-    const borderColor = maturingBalance.isZero() ? 'gray.450' : 'blue.200';
+    const bg = useColorModeValue('white', 'rgba(255, 255, 255, 0.06)');
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/NodeMaturingSection.tsx:37]: Here as well for using a color var.

File: apps/staking/src/components/node/NodeMaturingSection.tsx [hunk 6]
```diff
@@ -34,26 +34,18 @@ export const NodeMaturingSection: FC<INodeMaturingSection> = ({
...
 
     return (
         <Box
+            borderRadius={'2xl'}
             position="relative"
             bg={bg}
             p={6}
             pl={7}
-            border="1px solid var(--chakra-colors-gray-100)"
+            border="1px solid rgba(255, 255, 255, 0.10)"
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/NodeMaturingSection.tsx:47]: And here.

File: apps/staking/src/components/node/NodeReleasingSection.tsx [hunk 7]
```diff
@@ -37,27 +37,19 @@ export const NodeReleasingSection: FC<INodeReleasingSection> = ({
     releasingLeftShort,
     onWithdraw,
 }) => {
-    const bg = useColorModeValue('white', 'gray.800');
-    const borderColor = releasingBalance.isZero() ? 'gray.450' : 'blue.200';
+    const bg = useColorModeValue('white', 'rgba(255, 255, 255, 0.06)');
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/NodeReleasingSection.tsx:40]: And here.

File: apps/staking/src/components/node/NodeReleasingSection.tsx [hunk 8]
```diff
@@ -37,27 +37,19 @@ export const NodeReleasingSection: FC<INodeReleasingSection> = ({
...
     return (
         <Box
+            borderRadius={'2xl'}
             position="relative"
             bg={bg}
             p={6}
             pl={7}
             mt={5}
-            border="1px solid var(--chakra-colors-gray-100)"
+            border="1px solid rgba(255, 255, 255, 0.10)"
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/NodeReleasingSection.tsx:51]: Here as well.

File: apps/staking/src/components/node/NodeStakedBalanceSection.tsx [hunk 9]
```diff
@@ -31,17 +31,26 @@ export interface INodeStakedBalanceSection {
 export const NodeStakedBalanceSection: FC<INodeStakedBalanceSection> = ({
     stakedBalance,
 }) => {
-    const bg = useColorModeValue('white', 'gray.800');
-
+    const bg = useColorModeValue('white', 'rgba(255, 255, 255, 0.06)');
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/NodeStakedBalanceSection.tsx:34]: And here.

File: apps/staking/src/components/node/NodeStakedBalanceSection.tsx [hunk 10]
```diff
@@ -31,17 +31,26 @@ export interface INodeStakedBalanceSection {
...
     return (
-        <Box bg={bg} shadow="md" p={6} mt={5} minHeight="7.125rem">
+        <Box
+            bg={bg}
+            shadow="md"
+            p={6}
+            mt={5}
+            minHeight="7.125rem"
+            borderRadius={'2xl'}
+            border="1px solid rgba(255, 255, 255, 0.10)"
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/NodeStakedBalanceSection.tsx:45]: Here as well for color var.

File: apps/staking/src/components/node/modals/NodeAllowanceModal.tsx [hunk 11]
```diff
@@ -68,7 +70,12 @@ export const NodeAllowanceModal: FC<INodeAllowanceModalProps> = ({
                 initialFocusRef={inputFocusRef}
             >
                 <ModalOverlay />
-                <ModalContent>
+                <ModalContent
+                    bg={bgModal}
+                    border={'1px solid rgba(255, 255, 255, 0.20)'}
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/modals/NodeAllowanceModal.tsx:75]: Here as well for a color var.

File: apps/staking/src/components/node/modals/NodeBalanceModal.tsx [hunk 12]
```diff
@@ -97,7 +99,12 @@ export const NodeBalanceModal: FC<INodeBalanceModalProps> = ({
                 initialFocusRef={inputFocusRef}
             >
                 <ModalOverlay />
-                <ModalContent>
+                <ModalContent
+                    bg={bgModal}
+                    border={'1px solid rgba(255, 255, 255, 0.20)'}
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/modals/NodeBalanceModal.tsx:104]: Here as well for a color var.

File: apps/staking/src/components/node/modals/NodeRetireModal.tsx [hunk 13]
```diff
@@ -54,7 +56,12 @@ export const NodeRetireModal: FC<INodeRetireModalProps> = ({
     return (
         <Modal onClose={onClose} isOpen={isOpen} isCentered>
             <ModalOverlay />
-            <ModalContent>
+            <ModalContent
+                bg={bgModal}
+                border={'1px solid rgba(255, 255, 255, 0.20)'}
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/modals/NodeRetireModal.tsx:61]: Here as well for a color var.

File: apps/staking/src/components/node/modals/NodeStakeModal.tsx [hunk 14]
```diff
@@ -89,7 +91,12 @@ export const NodeStakeModal: FC<INodeStakeModalProps> = ({
                 initialFocusRef={inputFocusRef}
             >
                 <ModalOverlay />
-                <ModalContent>
+                <ModalContent
+                    bg={bgModal}
+                    border={'1px solid rgba(255, 255, 255, 0.20)'}
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/modals/NodeStakeModal.tsx:96]: Here as well.

File: apps/staking/src/components/node/modals/NodeUnstakeModal.tsx [hunk 15]
```diff
@@ -89,7 +91,12 @@ export const NodeUnstakeModal: FC<INodeUnstakeModalProps> = ({
                 initialFocusRef={inputFocusRef}
             >
                 <ModalOverlay />
-                <ModalContent>
+                <ModalContent
+                    bg={bgModal}
+                    border={'1px solid rgba(255, 255, 255, 0.20)'}
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/modals/NodeUnstakeModal.tsx:96]: Here as well for a color var.

File: apps/staking/src/components/node/steps/CustomizeEthereumNode.tsx [hunk 16]
```diff
@@ -33,15 +34,28 @@ const CustomizeEthereumNode = ({
     onStepActive,
 }: IStep) => {
     const [state, setState] = useStepState({ inFocus });
-    const thirdPartyColor = useColorModeValue('blue.500', 'blue.200');
-
+    const thirdPartyColor = useColorModeValue(
+        'light.primary',
+        'teal.secondary'
+    );
+    const bg = useColorModeValue('teal.light', 'rgba(255, 255, 255, 0.06)');
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/steps/CustomizeEthereumNode.tsx:41]: Here and below, please use a color var instead.

File: apps/staking/src/components/node/steps/CustomizeEthereumNode.tsx [hunk 17]
```diff
@@ -107,7 +127,7 @@ const CustomizeEthereumNode = ({
                 >
                     <Button
                         minWidth={{ base: '10rem' }}
-                        colorScheme="blue"
+                        colorScheme={colorMode === 'dark' ? 'cyan' : 'teal'}
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/steps/CustomizeEthereumNode.tsx:130]: You can use `useColorModeValue` for the color scheme as well.

File: apps/staking/src/components/node/steps/HireNode.tsx [hunk 18]
```diff
@@ -49,7 +56,7 @@ const HireNode = ({
     inFocus,
 }: IStep) => {
     const [, setNodeAddressAtom] = useAtom(hiredNodeAddressAtom);
-    const tipsBgColor = useColorModeValue('gray.80', 'gray.800');
+    const tipsBgColor = useColorModeValue('white', 'rgba(255, 255, 255, 0.20)');
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/steps/HireNode.tsx:59]: Same here for a color var.

File: apps/staking/src/components/node/steps/HireNode.tsx [hunk 19]
```diff
@@ -70,6 +77,12 @@ const HireNode = ({
                 : { ...state, [name]: validation };
         });
     };
+    const bg = useColorModeValue('teal.light', 'rgba(255, 255, 255, 0.06)');
+    const borderColor = useColorModeValue(
+        'light.grey.tertiary',
+        'rgba(255, 255, 255, 0.10)'
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/steps/HireNode.tsx:83]: And here.

File: apps/staking/src/components/node/steps/HireNode.tsx [hunk 20]
```diff
@@ -150,7 +169,7 @@ const HireNode = ({
                     <Button
                         disabled={!enableNext || node.transaction?.isOngoing}
                         isLoading={node.transaction?.isOngoing}
-                        colorScheme="blue"
+                        colorScheme={colorMode === 'dark' ? 'cyan' : 'teal'}
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/steps/HireNode.tsx:172]: Please use `useColorModeValue` for the color scheme.

File: apps/staking/src/components/node/steps/SetAllowance.tsx [hunk 21]
```diff
@@ -161,7 +162,12 @@ const SetAllowance = ({ stepNumber, inFocus, onStepActive }: IStep) => {
     const [stepState] = useStepState({ inFocus });
     const [allowanceAmount, setAllowance] = useState<string | null>();
     const [errors, setErrors] = useState<Errors>({});
-
+    const bg = useColorModeValue('teal.light', 'rgba(255, 255, 255, 0.06)');
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/steps/SetAllowance.tsx:165]: Here and below, please use a color var instead.

File: apps/staking/src/components/node/steps/SetAllowance.tsx [hunk 22]
```diff
@@ -219,7 +230,7 @@ const SetAllowance = ({ stepNumber, inFocus, onStepActive }: IStep) => {
                     <Button
                         disabled={!enableBtn}
                         minWidth={{ base: '10rem' }}
-                        colorScheme="blue"
+                        colorScheme={colorMode === 'dark' ? 'cyan' : 'teal'}
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/steps/SetAllowance.tsx:233]: Same as above, please use `useColorModeValue` instead.

File: apps/staking/src/components/node/steps/SetUpNode.tsx [hunk 23]
```diff
@@ -30,7 +31,7 @@ const { COMPLETED } = StepStatus;
 
 const CopyBoard = ({ command, children }) => {
     const { hasCopied, onCopy } = useClipboard(command);
-    const bgColor = useColorModeValue('gray.80', 'gray.800');
+    const bgColor = useColorModeValue('white', 'rgba(255, 255, 255, 0.20)');
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/steps/SetUpNode.tsx:34]: Here as well for a color var.

File: apps/staking/src/components/node/steps/SetUpNode.tsx [hunk 24]
```diff
@@ -73,23 +74,33 @@ const SetUpNode = ({
...
             </Text>{' '}
             --wallet /root/.ethereum/key --create --verbose;
         </>
     );
 
     const [state, setState] = useStepState({ inFocus });
-    const linkColor = useColorModeValue('blue.500', 'blue.200');
-
+    const linkColor = useColorModeValue('light.primary', 'teal.secondary');
+    const bg = useColorModeValue('teal.light', 'rgba(255, 255, 255, 0.06)');
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/steps/SetUpNode.tsx:86]: And here, as well as below. Please use a color var.

File: apps/staking/src/components/node/steps/SetUpNode.tsx [hunk 25]
```diff
@@ -176,7 +193,7 @@ const SetUpNode = ({
                         PREVIOUS
                     </Button>
                     <Button
-                        colorScheme="blue"
+                        colorScheme={colorMode === 'dark' ? 'cyan' : 'teal'}
```
Comment (nevendyulgerov) [inline apps/staking/src/components/node/steps/SetUpNode.tsx:196]: Please use `useColorModeValue` here.

File: apps/staking/src/components/stake/components/StakedBalanceSection.tsx [hunk 26]
```diff
@@ -47,8 +46,8 @@ export const StakedBalanceSection: FC<IStakedBalanceSection> = ({
             bg={bg}
             shadow={boxShadow}
             borderWidth="1px"
-            borderColor={borderColor}
-            borderRadius="1rem"
+            borderRadius={'2xl'}
+            border="1px solid rgba(255, 255, 255, 0.10)"
```
Comment (nevendyulgerov) [inline apps/staking/src/components/stake/components/StakedBalanceSection.tsx:50]: Here as well for a color var.

File: apps/staking/src/pages/node/[node]/manage.tsx [hunk 27]
```diff
@@ -102,8 +102,13 @@ const ManageNode: FC = () => {
...
 
     // dark mode support
-    const bg = useColorModeValue('gray.80', 'header');
-
+    const bg = useColorModeValue('white', 'dark.gray.quaternary');
+    const bgSection = useColorModeValue(
+        'light.gray.secondary',
+        'dark.gray.primary'
+    );
+    const bgBox = useColorModeValue('white', 'rgba(255, 255, 255, 0.06)');
```
Comment (nevendyulgerov) [inline apps/staking/src/pages/node/[node]/manage.tsx:110]: Here as well.

**Review by nevendyulgerov — APPROVED** [review]

**Pull request merged**